packages/feedback/src/index.ts

```diff
@@ -1 +1,433 @@
-export {sendFeedbackRequest} from './util/sendFeedbackRequest';
+import { getCurrentHub } from '@sentry/core';
+import type { Integration } from '@sentry/types';
+import { isNodeEnv } from '@sentry/utils';
+
+import type { FeedbackConfigurationWithDefaults } from './types';
+import { sendFeedbackRequest } from './util/sendFeedbackRequest';
+import { Dialog } from './widget/Dialog';
+import { Icon } from './widget/Icon';
+
+export { sendFeedbackRequest };
+
+type ElectronProcess = { type?: string };
+
+// Electron renderers with nodeIntegration enabled are detected as Node.js so we specifically test for them
+function isElectronNodeRenderer(): boolean {
+  return typeof process !== 'undefined' && (process as ElectronProcess).type === 'renderer';
+}
+/**
+ * Returns true if we are in the browser.
+ */
+function isBrowser(): boolean {
+  // eslint-disable-next-line no-restricted-globals
+  return typeof window !== 'undefined' && (!isNodeEnv() || isElectronNodeRenderer());
+}
```

Comment:
Temp, this is copy and pasted from replay -- should this be in @sentry/utils?

Reply:
I think we could move it to utils, yes!